DIRECTORY.md

```
@@ -4,18 +4,35 @@
     * [Accumulate](https://github.com/BrianLusina/gopher-land/blob/master/algorithms/accumulate/accumulate.go)
     * [Accumulate Test](https://github.com/BrianLusina/gopher-land/blob/master/algorithms/accumulate/accumulate_test.go)
   * Arrays
+    * Intersection
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)